### PR TITLE
Empty Catch Block Check, updated docs

### PR DESCRIPTION
**Pull request by alexkravin**
According to #571 - updated docs

### CODE REVIEW

File: src/xdocs/config_blocks.xml [hunk 1]
```diff
@@ -150,7 +151,7 @@
 &lt;/module&gt;
         </source>
         <p>
-          Such empty blocks would be both suppressed:
+          Such empty blocks would suppressed:
```
Comment (mkordas) [inline src/xdocs/config_blocks.xml]: Change to `would be suppressed`

**Comment by mkordas** [comment]
Looks nice, thanks :+1:

**Comment by alexkravin** [comment]
@mkordas,

you're right about "would <b>be</b> suppressed", just didn't notice, thanks, updated

**Comment by romani** [comment]
merged as FF

**Pull request closed without merging**